### PR TITLE
Fix auto-selected modules for online migrations

### PR DESCRIPTION
**Pull request by Sergeykot**

### CODE REVIEW

File: package/rmt-server.changes [hunk 1]
```diff
@@ -4,6 +4,7 @@ Tue Oct 30 15:35:17 UTC 2018 - skotov@suse.com
 - Version 1.0.7
 - Import migration_extra attribute from SCC API
 - Auto-selected modules feature migration engine changes
+- Restore old behavior for ASMM in online migrations
```
Comment (digitaltom) [inline package/rmt-server.changes]: I don't think we need this line. Maybe the line before can be worded more customer friendly: 
`Offline migrations will automatically add additional modules to the client systems depending on the base product`

File: spec/models/migration_engine_spec.rb [hunk 2]
```diff
@@ -435,4 +381,54 @@
...
+        ProductsExtensionsAssociation.find_by(
+          product: sle15,
+          extension: recommended_module,
+          root_product: sle15
+        ).update!(recommended: true)
+
+        recommended_module
+      end
+
+      it { is_expected.to contain_exactly([sle15, sle15_recommended_module]) }
```
Comment (digitaltom) [inline spec/models/migration_engine_spec.rb]: the context name (`...modules do _not_ get added...`) and the test do not match

File: spec/models/migration_engine_spec.rb [hunk 3]
```diff
@@ -435,4 +381,54 @@
...
+        ProductsExtensionsAssociation.find_by(
+          product: sle15,
+          extension: extra_module,
+          root_product: sle15
+        ).update!(migration_extra: true)
+
+        extra_module
+      end
+
+      it { is_expected.to contain_exactly([sle15, sle15_extra_module]) }
```
Comment (digitaltom) [inline spec/models/migration_engine_spec.rb]: the context name (`...do _not_ get added...`) and the test do not match

Comment (Sergeykot) [reply spec/models/migration_engine_spec.rb]: fixed

File: spec/models/migration_engine_spec.rb (hunk 2 above)
Comment (Sergeykot) [reply spec/models/migration_engine_spec.rb]: fixed

File: package/rmt-server.changes (hunk 1 above)
Comment (Sergeykot) [reply package/rmt-server.changes]: fixed

**Pull request merged**